source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,155 @@
 ---
 title: HTML coding style
-last_reviewed_on: 2016-02-03
+last_reviewed_on: 2019-06-20
 review_in: 6 months
 ---
 
 # <%= current_page.data.title %>
 
-## HTML5
+## Browser support
 
-* Always quote attribute values - the decision tree for remember when to
-  selectively quote is too much mental overhead, so just quote them
-* Always use well-formed HTML - implicitly closed elements such as
-  `<p>` cause unnecessary parsing overhead and can cause issues in
-  accessibility
+The GOV.UK Service Manual lists [browsers that it is required to support](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in). Specific
+services, such as GOV.UK Frontend, may have [stricter standards of browser
```

Comment:
GOV.UK Frontend is not a service, but is used to build services.

The guidance in the service manual sets a _minimum_ standard that services should adhere to in terms of browser support. GOV.UK Frontend has a different browser support matrix (in that we still support IE8 - 10) because we know that there are some services out there that still need to support those browsers (because it's still used by their users) and if GOV.UK Frontend didn't enable them to do that they would not be able to use it.

It might be worth thinking about what we want the user to do with this information. Why is it important to know that GOV.UK Frontend has a different browser support matrix? I don't think this is clear at the minute.

Reply:
Agreed 👍 I wanted a section on browser support because when I joined GOV.UK, I wasn't sure what the browser support expectations were for the service I was building.

Initially I was under the impression that the minimum support levels of the GOV.UK Frontend were what GOV.UK supported as a whole, then someone informed me about the service standard minimum support levels and how they differ.

I've now amended this section, hopefully it is now clearer. Want to make it clear that the service manual defines the minimum browser support a service should adhere to, but that if it uses, say, the GOV.UK Frontend, any PRs to that must continue to be compatible with its own minimum standards.